### PR TITLE
Port to systemD so we can use Ubuntu Wily based AMIs.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: content-api-itunes-rss.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=itunes-rss
+
+[Service]
+User=content-api
+Group=content-api
+Restart=no
+Environment='HOME=/home/content-api'
+Environment='JAVA_OPTS=-Xmx256m -Xms256m -XX:+UseConcMarkSweepGC -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:/home/content-api/logs/gc.log -XX:+TieredCompilation -XX:ReservedCodeCacheSize=256m -Dconfig.file=/etc/gu/content-api-itunes-rss.conf'
```
Comment (mchv) [inline content-api-itunes-rss.service]: if not using `nano` we should increase the memory to at least `1024m`

**Comment by oilnam** [comment]
Hey @LATaylor-guardian what happened to this one?

**Comment by LATaylor-guardian** [comment]
@oilnam It just got forgotten about it. Let's leave for now as I don't fancy breaking this before the weekend!

**Comment by mchv** [comment]
Thanks, it has been useful for #38 !

**Comment by LATaylor-guardian** [comment]
Ah, completely forgot I even done this!

**Pull request closed without merging**